src/api/rootResolver.js

```diff
@@ -24,10 +56,39 @@ const resolvers = {
     },
     async parties() {
       return eventsList.map(event => ({ ...event, __typename: 'PartyMeta' }))
+    },
+    async events() {
+      return (await getEvents()).map((event)=>{
+        console.log('event', event)
+        return {
+          name: event.args.deployedAddress,
+          address: event.args.deployedAddress,
+          __typename: event.event
+        }
+      })
     }
   },
 
   Mutation: {
```

Comment:
We can leave this here for now until we find a better resolver to put this in, but we'll probably also put this in a different resolver in the future

Reply:
Same as above

Reply:
Fine with this too